src/components/SpruceForm/errors.ts

```diff
@@ -74,7 +74,8 @@ export const transformErrors = (errors: AjvError[]) =>
             case "validSlack":
```

Comment:
Because we have this function `validSlack`, I don't think users will actually be able to submit a subscription using a Slack member ID. Do you want to try fixing it in this ticket (which will require updating the regex expression found [here](https://github.com/evergreen-ci/spruce/blob/2a281b272efeb91c38b653e4ccbc1e63b1266835/src/utils/validators/index.ts#L46))? Or if you want to do it separately, that's ok too :)

Reply:
good point, I'll create a ticket for it.